app/services/deploy_service.rb

```diff
@@ -100,9 +106,9 @@ def send_slack_alert(pull_request, webhook_url, merge_at)
       uri = URI.parse webhook_url
       request = Net::HTTP::Post.new(uri.request_uri)
       request['Content-Type'] = 'application/json'
+      distance_of_time = merge_at.past? ? 'a few minutes' : distance_of_time_in_words_to_now(merge_at)
```

Comment:
I've reordered the notifying and merging steps so that notifying always happens at least once. (Previously, if a deploy PR was already open when a project enabled merges, it would be merged with no notification.) However, this means that `merge_at` might be in the past. Instead of saying "...will be released X hours ago," we should say "...will be released in a few minutes." (The merge will happen on the next cron run after the notification.)